app/api/endpoint.py

```
@@ -32,6 +38,31 @@ def recommend_on_demand(request: Request):
+                #      AND combined.brand_id = rbd.brand_id
+                #      AND rbd.data_type = 'EXCLUDE'
+                #     WHERE rbd.id IS NULL
+                #       AND combined.user_id = {user_id}
+                # """
+                # exclude_brand_ids = pd.read_sql(exclude_query, conn)["brand_id"].tolist()
+                # brand_df = brand_df[~brand_df["brand_id"].isin(exclude_brand_ids)]
+                # interaction_df = interaction_df[~interaction_df["brand_id"].isin(exclude_brand_ids)]
+                # user_brand_df = user_brand_df[~user_brand_df["brand_id"].isin(exclude_brand_ids)]
+
+                exclude_brand_df = load_exclude_brands(conn, user_ids=[user_id])
+                exclude_brand_ids = set(exclude_brand_df["brand_id"].tolist())
```

Ruff (0.12.2)

Local variable `exclude_brand_ids` is assigned to but never used

Remove assignment to unused variable `exclude_brand_ids`

(F841)